### PR TITLE
1.8.1 Changelog

### PR DESCRIPTION
**Pull request by jamiedemaria**

### CODE REVIEW

**Comment by jamiedemaria** [comment]
* **#23673** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/23673?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @jamiedemaria and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,57 @@
...
+- [dagster-aws] Glue jobs launched via pipes can be automatically canceled if Dagster receives a termination signal.
+- [dagster-databricks] `dagster-databricks` now supports `databricks-sdk<=0.17.0`.
+- [dagster-dbt] The `DagsterDbtTranslator` class now supports a modfiying the `AutomationCondition` for dbt models by overriding `get_automation_condition`.
+- [dagster-pandera] `dagster-pandera` now supports `polars`.
+- [dagster-sdf] Table and columns tests can now be used as asset checks.
+- [dagster-embedded-elt] Column metadata and lineage can be fetched on Sling assets by chaining the new `replicate(...).fetch_column_metadata()` method.
+- [dagster-embedded-elt] dlt resource docstrings will now be used to populate asset descriptions, by default.
+- [dagster-embedded-elt] dlt assets now generate column metadata.
+- [dagster-embedded-elt] dlt transformers now refer to the base resource as upstream asset.
+- [ui] Replaced GraphQL Explorer with GraphiQL.
```
Comment (sryza) [inline CHANGES.md:20]: We generally put the [ui] stuff before the integrations.

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,57 @@
...
+- [ui] The run timeline can now be grouped by job or by automation.
+- [ui] For users in the experimental navigation flag, schedules and sensors are now in a single merged automations table.
+- [ui] Logs can now be filtered by metadata keys and values.
+- [ui] Logs for `RUN_CANCELED` events now display relevant error messages.
+- [community-contribution][dagster-azure] `AzureBlobComputeLogManager` now supports service principals, thanks @[ion-elgreco](https://github.com/ion-elgreco)!
+- [community-contribution][dagster-datahub] `dagster-datahub` now allows pydanitc versions below 3.0.0, thanks @[kevin-longe-unmind](https://github.com/kevin-longe-unmind)!
+- [community-contribution][dagster-openai] `OpenAIResource` now supports `organization`, `project` and `base_url` for configurting the OpenAI client, thanks @[**chasleslr](https://github.com/chasleslr)!
```
Comment (sryza) [inline CHANGES.md:27]: are the asterixes here on purpose?

File: CHANGES.md [hunk 3]
```diff
@@ -1,5 +1,57 @@
...
+- [dagster-embedded-elt] dlt resource docstrings will now be used to populate asset descriptions, by default.
+- [dagster-embedded-elt] dlt assets now generate column metadata.
+- [dagster-embedded-elt] dlt transformers now refer to the base resource as upstream asset.
+- [ui] Replaced GraphQL Explorer with GraphiQL.
+- [ui] The run timeline can now be grouped by job or by automation.
+- [ui] For users in the experimental navigation flag, schedules and sensors are now in a single merged automations table.
+- [ui] Logs can now be filtered by metadata keys and values.
+- [ui] Logs for `RUN_CANCELED` events now display relevant error messages.
+- [community-contribution][dagster-azure] `AzureBlobComputeLogManager` now supports service principals, thanks @[ion-elgreco](https://github.com/ion-elgreco)!
+- [community-contribution][dagster-datahub] `dagster-datahub` now allows pydanitc versions below 3.0.0, thanks @[kevin-longe-unmind](https://github.com/kevin-longe-unmind)!
```
Comment (sryza) [inline CHANGES.md:26]:
```suggestion
- [community-contribution][dagster-datahub] `dagster-datahub` now allows pydantic versions below 3.0.0, thanks @[kevin-longe-unmind](https://github.com/kevin-longe-unmind)!
```

File: CHANGES.md [hunk 4]
```diff
@@ -1,5 +1,57 @@
...
+- [dagster-embedded-elt] Column metadata and lineage can be fetched on Sling assets by chaining the new `replicate(...).fetch_column_metadata()` method.
+- [dagster-embedded-elt] dlt resource docstrings will now be used to populate asset descriptions, by default.
+- [dagster-embedded-elt] dlt assets now generate column metadata.
+- [dagster-embedded-elt] dlt transformers now refer to the base resource as upstream asset.
+- [ui] Replaced GraphQL Explorer with GraphiQL.
+- [ui] The run timeline can now be grouped by job or by automation.
+- [ui] For users in the experimental navigation flag, schedules and sensors are now in a single merged automations table.
+- [ui] Logs can now be filtered by metadata keys and values.
+- [ui] Logs for `RUN_CANCELED` events now display relevant error messages.
+- [community-contribution][dagster-azure] `AzureBlobComputeLogManager` now supports service principals, thanks @[ion-elgreco](https://github.com/ion-elgreco)!
```
Comment (sryza) [inline CHANGES.md:25]: In the past, we haven't included this "[community-contribution]" prefix, so I think we should leave it out for consistency.

**Review by sryza — APPROVED** [review]
Looks good after fixing a few nitpicks!

File: CHANGES.md (hunk 2 above)
Comment (jamiedemaria) [reply CHANGES.md:27]: nope, notion auto-bolded contributors names and i missed deleting this set of *

File: CHANGES.md (hunk 4 above)
Comment (jamiedemaria) [reply CHANGES.md:25]: definitely can delete, but the notion doc says to include the prefix, so we should make sure to update the instructions or figure out why they were added

**Pull request merged**